packages/nimble-components/src/checkbox/template.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-restricted-syntax */
+import { html, slotted, type ViewTemplate } from '@microsoft/fast-element';
+import type {
+    CheckboxOptions,
+    FoundationElementTemplate
+} from '@microsoft/fast-foundation';
+import type { Checkbox } from '.';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<Checkbox>,
+CheckboxOptions
+> = (_context, definition) => html`
+    <template
+        role="checkbox"
+        aria-checked="${x => x.checked}"
+        aria-required="${x => x.required}"
+        aria-disabled="${x => x.disabled}"
+        aria-readonly="${x => x.readOnly}"
+        tabindex="${x => (x.disabled ? null : x.tabIndex ?? 0)}"
```

Comment:
Can you add parenthesis here so it's less ambiguous? I can't tell if the behavior is
`(x.disabled ? null : x.tabIndex) ?? 0`
or
`x.disabled ? null : (x.tabIndex ?? 0)`

Reply:
Coupled with the other comment remove the eslint disable and move the logic to a method which has no issues with the `??` syntax.

Reply:
Prettier won't let me add the parens (or otherwise format the logic with newlines to make precedence clearer). I could:
1. leave it as `return this.disabled ? null : this.tabIndex ?? 0;`
2. add the parens and a `// prettier-ignore`
3. split it into `if (this.disabled) { return null; } return this.tabIndex ?? 0;`

I went with 2, but let me know if you prefer otherwise.